src/lpython/semantics/python_attribute_eval.h

```diff
@@ -59,6 +62,16 @@ struct AttributeHandler {
         }
     }
 
+    static ASR::asr_t* eval_int_bit_length(ASR::expr_t *s, Allocator &al, const Location &loc,
+            Vec<ASR::expr_t*> &args, diag::Diagnostics &/*diag*/) {
+        if (args.size() != 0) {
+            throw SemanticError("bit_length() takes no arguments (" + std::to_string(args.size()) + " given)", loc);
+        }
+        ASR::ttype_t *int_type = ASRUtils::TYPE(ASR::make_Integer_t(al, loc,
+                                        4, nullptr, 0));
+        return ASR::make_IntegerBitLen_t(al, loc, s, int_type, nullptr);
```

Comment:
Instead of passing a `nullptr`, pass the compile-time `value`.

Reply:
I think you forgot to address this comment? @Abdelrahman-Kh-Fouad

Reply:
i don't know what `value` to pass